src/api/utils/notifications/email_notices.py

```diff
@@ -46,29 +47,43 @@ def send_email(**kwargs):
     :kwarg recipients:
     :return bool:
     """
-    mail_gun_url = current_app.config["MAIL_GUN_URL"]
-    mail_gun_api_key = current_app.config["MAIL_GUN_API_KEY"]
-    mail_gun_test = current_app.config.get("MAIL_GUN_TEST")
-
-    if kwargs["sender"] == "":
+    if not kwargs["sender"] or not kwargs["sender"].strip():
```

Comment:
First condition should be `kwargs.get("sender")` to prevent raising a KeyError if sender isn't in the kwargs.